packages/amplify-e2e-tests/src/amplify-app-helpers/amplify-app-validation.ts

```diff
@@ -1,41 +1,43 @@
 import * as fs from 'fs-extra';
 import * as path from 'path';
-import { pathManager } from 'amplify-cli-core';
+import { AmplifyFrontend, pathManager } from 'amplify-cli-core';
```

Comment:
Does this import need to be updated?

```suggestion
import { AmplifyFrontend, pathManager } from '@aws-amplify/amplify-cli-core';
```

Reply:
oups, had to revert, looks that amplify-cli-core wasn't yet migrated

Reply:
committed, this looks like it required some additional changes

Reply:
nevermind,looks that we have 2 packages, one with @aws-amplify and one without